docs/components/table.md

```diff
@@ -209,6 +209,82 @@ various parts of the table.
 <KTable @row:click="rowHandler" @row:dblclick="rowHandler">
 ```
 
+- The rows can be clicked and metakeys are also supported in the row click handler. Hyperlinks and Buttons can also be clicked separately, as in the below example:
+
+<KTable
+  :options="tableOptionsLink"
+  isClickable
+  @row:click="handleRowClick">
+  <template v-slot:company="{rowValue}">
+    <a v-if="rowValue" @click="linkHander">{{rowValue.label}}</a>
+    <span v-else>{{rowValue}}</span>
+  </template>
+  <template v-slot:actions><KButton appearance="secondary" @click="buttonHandler">Visit Website</KButton></template>
+</KTable>
+
+```vue
+<KTable
+  :options="tableOptionsLink"
+  isClickable
+  @row:click="handleRowClick">
+  <template v-slot:company="{rowValue}">
+    <a v-if="rowValue" @click="linkHander">{{rowValue.label}}</a>
+    <span v-else>{{rowValue}}</span>
+  </template>
+  <template v-slot:actions><KButton appearance="secondary" @click="buttonHandler">Visit Website</KButton></template>
+</KTable>
+<script>
+import { defaultSorter } from '@kongponents/KTable'
+export default {
+  data() {
+    return {
+      row: null,
+      eventType: '',
+      tableOptions: {
+        headers: [
+          { label: 'Company', key: 'company' }
+        ],
+        data: [
```

Comment:
this data doesn't look like anything we'd pass in, can you update with a more common set of items?

Reply:
I thought this was just meant to be an example. What is a more 'common' set of items? we just need an example with a set of data with clickable links right?